src/components/custom/_agreement-label.js

```diff
@@ -16,12 +16,16 @@ const AgreementLabel = ({
     handleChangeCheckbox,
     isChecked,
     color,
-    link_text = localize('I agree to the <0>terms and conditions</0>'),
+    pep_label,
+    link_text = localize(pep_label || 'I agree to the <0>terms and conditions</0>'),
```

Comment:
```suggestion
    link_text = pep_label || localize('I agree to the <0>terms and conditions</0>'),
```
Shouldn't `pep_label` be out of the localize since it itself is already wrapped in `localize`?

Reply:
There are a few other instances of this in the PR.

Reply:
Refactored here.